go/vt/topotools/events/migrate_syslog.go

```diff
@@ -52,10 +52,16 @@ func (ev *MigrateServedTypes) Syslog() (syslog.Priority, string) {
 
 	sourceShards := make([]string, len(ev.SourceShards))
 	for i, shard := range ev.SourceShards {
+		if sourceshards[i] == nil {
```

Comment:
Shouldn't you check if `shard == nil`?

Reply:
I agree, I think this and below should be:
if shard != nil {
  sourceShards[i] = shard.ShardName()
}

Reply:
Oops! Re-fix incoming.